### PR TITLE
Create the ES only in the cloud

### PR DESCRIPTION
**Pull request by Avilir**
Since hit some problems with Port-FWD to ES server on BM cluster, and the Internal ES server is needed only in the cloud environment, it will be created only in the cloud.

### CODE REVIEW

File: tests/e2e/performance/test_fio_benchmark.py [hunk 1]
```diff
@@ -161,26 +168,25 @@ def test_fio_workload_simple(self, ripsaw, es, interface, io_pattern):
             'resources/crds/'
             'ripsaw_v1alpha1_ripsaw_crd.yaml'
         )
-        sc = 'ocs-storagecluster-ceph-rbd' if interface == 'CephBlockPool' else 'ocs-storagecluster-cephfs'
+        if interface == 'CephBlockPool':
+            sc = 'ocs-storagecluster-ceph-rbd'
```
Comment (ebenahar) [inline tests/e2e/performance/test_fio_benchmark.py:172]: Please save 'ocs-storagecluster-ceph-rbd' in constants.py

File: tests/e2e/performance/test_fio_benchmark.py [hunk 2]
```diff
@@ -161,26 +168,25 @@ def test_fio_workload_simple(self, ripsaw, es, interface, io_pattern):
...
+        else:
+            sc = 'ocs-storagecluster-cephfs'
```
Comment (ebenahar) [inline tests/e2e/performance/test_fio_benchmark.py:174]: Please save 'ocs-storagecluster-cephfs' in constants.py

File: tests/e2e/performance/test_fio_benchmark.py (hunk 1 above)
Comment (Avilir) [reply tests/e2e/performance/test_fio_benchmark.py]: done

File: tests/e2e/performance/test_fio_benchmark.py (hunk 2 above)
Comment (Avilir) [reply tests/e2e/performance/test_fio_benchmark.py]: done

File: tests/e2e/performance/test_fio_benchmark.py [hunk 3]
```diff
@@ -24,10 +24,17 @@
 
 @pytest.fixture(scope='function')
 def es(request):
+
+    # Create internal ES only if Cloud platform is tested
+    if node.get_provider().lower() in constants.CLOUD_PLATFORMS:
+        es = ElasticSearch()
+    else:
+        es = None
```
[the remark this answers is not in the document]
Comment (Avilir) [reply tests/e2e/performance/test_fio_benchmark.py:32]: i think in this way it is more readable

[the remark this answers is not in the document]
Comment (ebenahar) [reply tests/e2e/performance/test_fio_benchmark.py:32]: ok

**Pull request merged**